misc/io-internal/src/main/ballerina/print.bal

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/jballerina.java;
+
+# Prints `any|error` value to the STDOUT.
+#
+# + value - The value to be printed
+public isolated function println(any|error value) {
```

Comment:
For this module to be compatible with `ballerina/io`, this function should have the same signature?

https://lib.ballerina.io/ballerina/io/0.6.0-beta.2/functions#println